ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py

```diff
@@ -1507,6 +1507,39 @@ def test_jax_numpy_power(
     )
 
 
+# arange
+@handle_cmd_line_args
+@given(
+    dtype_and_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"), num_arrays=3, shape=(1,), shared_dtype=True, min_value=0, max_value=100,
```

Comment:
We don't need to specify the range you can remove the `max_value` and `min_value`